reana_workflow_controller/rest/workflows.py

```diff
@@ -237,7 +247,19 @@ def get_workflows(paginate=None):  # noqa
                 "status": workflow.status.name,
                 "user": user_uuid,
                 "created": workflow.created.strftime(WORKFLOW_TIME_FORMAT),
-                "progress": get_workflow_progress(workflow),
+                "run_started_at": (
+                    workflow.run_started_at.strftime(WORKFLOW_TIME_FORMAT)
+                    if workflow.run_started_at
+                    else None
+                ),
+                "run_finished_at": (
+                    workflow.run_finished_at.strftime(WORKFLOW_TIME_FORMAT)
+                    if workflow.run_finished_at
+                    else None
+                ),
```

Comment:
These two keys used to be under `progress` as far as I understood. So it means that the API response format changes, couldn't this have some unforeseen incompatibilities in the clients (r-client, r-ui, external clients (lukas), etc.)?

Reply:
I have updated the PR, now both the keys are in `progress` which will not have API response format mismatch.